eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     # Linux arm64
     - ${{ if eq(parameters.platform, 'linux_arm64') }}:
       - ${{ if or(eq(parameters.jobParameters.isExtraPlatformsBuild, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - (Ubuntu.2510.ArmArch.Open)Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-25.10-helix-arm64v8
+        - (Ubuntu.2604.ArmArch.Open)AzureLinux.3.Arm64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-26.04-helix-arm64v8
```

Comment:
The main thing here is that (I think) we're going to be moving the kernel to the AzLinux version, rather than Ubuntu. Even though we're using an ubuntu container, the kernel is going to be picked up from the base.

I think that's OK, but we've learned in the past that these kernel versions do have potential impact. Do we need some other coverage of the Ubuntu kernel somewhere?

Also, I see we've already made this change for x64, so it's clearly not high risk to do the same for arm64.